entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Wait for MySQL to be ready
+echo "Waiting for MySQL..."
+until nc -z db 3306; do
+  sleep 3
+done
+echo "MySQL is up - continuing..."
+
+# Run migrations and start the server
+python manage.py migrate
+python manage.py runserver 0.0.0.0:8000
```

Comment:
Ensure that the `manage.py` script and the necessary Django settings are correctly configured to connect to the database using the environment variables provided in the `docker-compose.yml` file.